core/src/main/java/com/scalar/db/sql/SqlStatementSession.java

```diff
@@ -0,0 +1,27 @@
+package com.scalar.db.sql;
+
+import com.scalar.db.sql.statement.Statement;
+
+public interface SqlStatementSession {
+  void begin();
+
+  void join(String transactionId);
```

Comment:
Also renamed `beginTransaction()` to `begin()` and `joinTransaction()` to `join()`.